src/App.js

```diff
@@ -30,10 +30,13 @@ const App = () => {
         <Route path="/success/:jwt" component={Success} />
         <PrivateRoute path="/onboarding" component={OnboardingPage} />
         <PrivateRoute path="/post/:id" component={Post} />
-        <PrivateRoute path="/room/:id/page/:page" component={SingleRoomDashboard}/>
+        <PrivateRoute
+          path="/room/:id/page/:page"
+          component={SingleRoomDashboard}
+        />
         <PrivateRoute path="/full-search" component={FullSearchResults} />
         <PrivateRoute path="/settings" component={Settings} />
-        <PrivateRoute path="/admin-settings" component={AdminSettings} />
+        {/* <PrivateRoute path="/admin-settings" component={AdminSettings} /> */}
```

Comment:
This is commented out do we still need it?

Reply:
No we don't but I'm leaving App.js till the end after we finished the migration of all components. We would need to restructure the components and delete the old components.